api/src/main/resources/liquibase.xml

```diff
@@ -600,6 +600,15 @@ VALUES
 			('CCLAD Cohort Type','2021-08-23',1,'5b7136fa-d207-4229-94a8-da6661ae00bf');
 		</sql>
 	</changeSet>
+	<changeSet id="ugandaemr-01032022-1401" author="dbaluku">
+		<comment>
+			Updating encounters and obs that were created with location_id set a null
```

Comment:
@dbaluku In the comment add the name of the location with uuid 86863db4-6101-4ecf-9a86-5e716d6504e4

Reply:
@ssmusoke  done